### PR TITLE
feat: Show Order Details

### LINKED ISSUE #1101: Show Orders Detail
### Feature Request

**Feature Description**

The organizer should be able to view Orders Detail for every item in Orders List. The UI should be kept similar to EventBrite as shown below.

orders_detail

   
**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #1101 

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [ ] My branch is up-to-date with the Upstream development branch.

Changes: 
1. Resolved minor bugs in Order List Layout.
2. Showing Details of Orders in OrderDetailFragment.
 
Screenshots for the change:
20180620224144-7c011ad592 gif-2-mp4 com

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/app/data/order/Order.java [hunk 1]
```diff
@@ -17,15 +17,20 @@
...
 
 @Data
-@Type("order")
 @Builder
+@Type("order")
 @AllArgsConstructor
+@ToString(exclude = {"event"})
 @JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
 @Table(database = OrgaDatabase.class, allFields = true)
+@EqualsAndHashCode()
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/data/order/Order.java]: Shouldn't be needed

**Comment by iamareebjamal** [comment]
Seems like order detail is loading from API rather than db. What's happening?

**Comment by Masquerade0097** [comment]
@iamareebjamal Fixed it. 
Just saw another issue. On backPresesed in OrderDetailFragment, OdersfRagment opens and closes instantly and user lands again in OrdeDetailFragment. Working on it.

**Comment by Masquerade0097** [comment]
Did the changes.

**Pull request merged**